apps/ledger-live-mobile/src/analytics/SegmentSetup.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useCallback } from "react";
+
+import { start } from "./segment";
+import type { AppStore } from "../reducers";
+import { useStore } from "react-redux";
+
+const SegmentSetup = (): null => {
+  const store = useStore();
+  const loadSegment = useCallback(async (store: AppStore) => {
+    try {
+      await start(store);
```

Comment:
Why don't we use `hooks` instead of functions ? I've seen that it was already like that before but just asking about the rationale behind that.